lib/content-services/src/lib/aspect-list/services/aspect-list.service.spec.ts

```diff
@@ -38,74 +43,155 @@ describe('AspectListService', () => {
 
     beforeEach(() => {
         TestBed.configureTestingModule({
-            imports: [HttpClientTestingModule]
+            providers: [provideHttpClient(), provideHttpClientTesting()]
         });
 
         aspectListService = TestBed.inject(AspectListService);
         apiService = TestBed.inject(AlfrescoApiService);
         aspectsApi = new AspectsApi(apiService.getInstance());
         spyOnProperty(aspectListService, 'aspectsApi', 'get').and.returnValue(aspectsApi);
+        standardAspectPagingMock = { list: { entries: [stdAspect1, stdAspect2, stdAspect3] } };
+        customAspectPagingMock = { list: { entries: [cstAspect1, cstAspect2, cstAspect3] } };
     });
 
-    it('should get one standard aspect', (done) => {
-        const visibleAspects = ['std:standardAspectOne'];
-        spyOn(aspectsApi, 'listAspects').and.returnValue(Promise.resolve(standardAspectPagingMock));
-        aspectListService.getStandardAspects(visibleAspects).subscribe((response) => {
-            expect(response).toEqual([stdAspect1]);
-            done();
+    describe('When API returns error', () => {
+        const visibleAspects = [];
```

Comment:
Please set type (based on usages it should be string[])

Reply:
Added